src/frequenz/sdk/microgrid/_data_pipeline.py

```diff
@@ -506,9 +506,9 @@ def frequency() -> GridFrequency:
     return _get().frequency()
 
 
-def voltage() -> VoltageStreamer:
+def voltage_per_phase() -> VoltageStreamer:
     """Return the 3-phase voltage measuring point."""
```

Comment:
what about `per-phase` instead of `3-phase`?

Reply:
fixed.